pootle/core/search/backends/elasticsearch.py

```diff
@@ -75,6 +92,10 @@ def search(self, unit):
             }
         )
 
+        # Sometimes urllib seems to return empty string so handle this here
```

Comment:
Isn't this masking the actual issue in the connection layer of ES? What do the logs show when the response is an empty string? This looks to me more like a workaround that silences an error.